### ISSUE TITLE
Tag name unique requirement across activities?

### ISSUE THREAD

**Issue by olejorgenb** [Razeeman/Android-SimpleTimeTracker#241]
When trying to create a new tag for activity A, I get "Name already used". I have used this name for a tag on another activity.  But since tags seems to be per activity this seems like an uneasonable restriction

**Comment by Mehlsine** [comment]
Nice observation. You could do this in previous versions, the logic must have been changed recently.

**Comment by olejorgenb** [comment]
Personally I think I would prefer having global tags, but with a UI to decide which tags being relevant for each activity.

I.e. all tags can be applied to all activities, but you would need to configure the tag as relevant to the activity to prevent the UI for tag selection from becoming cluttered.

**Comment by olejorgenb** [comment]
Ah, that's how it works :facepalm:  

When you create a tag you can assign the possible activity types. It's also possible to change this later by going to an activity associated with the tag and long-pressing the tag.

I think the UI should ask if you want to associate the existing tag with the activity type when you try to add a new tag with an existing tag to an activity. It would be nice if in addition to "Untagged" and "Add record tag", a third option "Add existing tag" would be a added that listed all tags.

**Comment by Razeeman** [comment]
Hello! Thank you for the feedback. Adding third option would be a nice addition, will try to look into it!

On a side note, "Name already used" message does not prevent creation of activity with the same name, it's more like the information to the user. Probably will change the color of this message from red to something else, to avoid misleading. Thank you for pointing this out.

**Comment by olejorgenb** [comment]
Another small semi-related thing I've noticed: On my phone when using autocomplete, the keyboard inserts a space after the word. This caused me to have a bunch of activities with a trailing space in their name. (might be tricky to do now, but ideally the trailing space would be stripped IMO). (I noticed this due do the the name already used message, which disappeared after deleting the trailing space)

---

Oh, and I really like the app btw :star_struck: 

---

I also since found the global "category an tag management page", which make things much less cumbersome
